src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx

```diff
@@ -198,6 +211,12 @@ export class ProsemirrorAdapter {
         });
         this.view.dispatch(transaction);
 
-        this.view.focus();
+        this.setFocus();
+    }
+
+    public setFocus() {
+        if (this.view) {
+            this.view.focus();
```

Comment:
We could also use `this.view?.focus()` to keep it short

Reply:
Updated :)